numba/tests/test_auto_constants.py

```diff
@@ -27,7 +29,12 @@ def pyfunc():
 
         self.assertEqual(pyfunc(), cfunc())
 
+    def test_module_string_constant(self):
+        @njit
+        def f():
+            return uc._GLOBAL_STR
+        self.assertEquals(f(), f.py_func())
```

Comment:
Also, shouldn't this be `assertEqual`? `assertEquals` is a deprecated alias https://docs.python.org/3/library/unittest.html#deprecated-aliases